k4e-agent-build

```diff
@@ -0,0 +1,5 @@
+sudo GOPROXY=proxy.golang.org,direct PWD=$PWD spec=$PWD outdir=$PWD make -f .copr/Makefile srpm
```

Comment:
I wonder why not to add code to top level [Makefile](https://github.com/jakub-dzon/yggdrasil/blob/main/Makefile)?

Reply:
I don't know what will be at the end. I expect that at the end the official Yggdrasil will contain our code so this will be deleted.

Reply:
OK, let's have it like this for now and improve when pushing the change to `Yggdrasil`.